src/DOM/File/File.purs

```diff
@@ -0,0 +1,18 @@
+module DOM.File.File
+  ( toBlob
+  , name
+  , lastModified
+  , lastModifiedDate
+  ) where
+
+import DOM.File.Types (File(), Blob())
+
+import Unsafe.Coerce (unsafeCoerce)
+import Data.Date (JSDate())
+
+toBlob :: File -> Blob
+toBlob = unsafeCoerce
```

Comment:
Oh no... this is the last suggestion, I promise! :wink:

I think this should probably be `fileToBlob` and in the `Types` module instead, as the coercions were done that way for the other types in this library.